packages/driver/cypress/e2e/commands/sessions/sessions.cy.ts

```diff
@@ -99,6 +99,7 @@ describe('cy.session', { retries: 0 }, () => {
       it('clears the browser cookie after each run', () => {
         cy.window()
         .then((win) => {
+          // @ts-expect-error TODO: cookie doesn't exist on win, usually cookie is set on document
           win.cookie = 'key=value; SameSite=Strict; Secure; Path=/fixtures'
         })
```

Comment:
Is this even valid?

Reply:
I don't think so. pretty sure this is supported to be `document.cookie`

Reply:
I think the whole test may need to be refactored in that case.